README.md

```diff
@@ -1,61 +1,46 @@
-
-## What's Yomitan?<!-- omit from toc -->
-
-Yomitan turns your web browser into a tool for building Japanese language literacy by helping you to decipher texts
-which would be otherwise too difficult to tackle. This extension is similar to [10ten Japanese Reader (formerly Rikaichamp)](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/) for Firefox and [Rikaikun](https://chrome.google.com/webstore/detail/rikaikun/jipdnfibhldikgcjhfnomkfpcebammhp) for Chrome, but it stands apart in its goal of being an all-encompassing learning tool as opposed to a mere browser-based dictionary.
+This extension is similar to [10ten Japanese Reader (formerly Rikaichamp)](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/) for Firefox and [Rikaikun](https://chrome.google.com/webstore/detail/rikaikun/jipdnfibhldikgcjhfnomkfpcebammhp) for Chrome.
 
 Yomitan provides advanced features not available in other browser-based dictionaries:
 
-- Interactive popup definition window for displaying search results.
-- On-demand audio playback for select dictionary definitions.
-- Kanji stroke order diagrams are just a click away for most characters.
-- Custom search page for easily executing custom search queries.
-- Support for multiple dictionary formats including [EPWING](https://ja.wikipedia.org/wiki/EPWING) via the [Yomitan Import](https://github.com/themoeway/yomitan-import) tool.
-- Automatic note creation for the [Anki](https://apps.ankiweb.net/) flashcard program via the [AnkiConnect](https://foosoft.net/projects/anki-connect) plugin.
-- Clean, modern code makes it easy for developers to [contribute](https://github.com/themoeway/yomitan/blob/master/CONTRIBUTING.md) new features.
+- 📈 Interactive popup definition window for displaying search results.
+- 🔊 Built-in audio playback for select Japanese words.
+- ✍ Kanji stroke order diagrams are generally just a click away.
+- 📝 [Automatic note creation](./docs/anki-integration.md#anki-integration) for the [Anki](https://apps.ankiweb.net/) flashcard program via the [AnkiConnect](https://foosoft.net/projects/anki-connect) plugin.
+- 🔍 Custom search page for easily executing custom search queries.
+- 📖 Support for multiple dictionary formats including [EPWING](https://ja.wikipedia.org/wiki/EPWING) via the [Yomitan Import](https://github.com/themoeway/yomitan-import) tool.
+- ✨ Clean, modern code makes it easy for developers to [contribute](#contributing) new features.
 
 [![Term definitions](img/ss-terms-thumb.png)](img/ss-terms.png)
 [![Kanji information](img/ss-kanji-thumb.png)](img/ss-kanji.png)
 [![Dictionary options](img/ss-dictionaries-thumb.png)](img/ss-dictionaries.png)
 [![Anki options](img/ss-anki-thumb.png)](img/ss-anki.png)
 
-## Helpful information<!-- omit from toc -->
+## Helpful information
 
-- [Migrating from Yomichan (legacy)](./docs/yomichan-migration.md#migrating-from-yomichan)
```

Comment:
Why remove?

Reply:
I moved it to the top

Reply:
![2024-05-19_18-54-13](https://github.com/themoeway/yomitan/assets/17340496/50914f77-0079-4a12-af15-7a88169d3e98)

Dictionaries seem to be gone, and I think it's useful to have the links clearly visible as a toc like that.

Reply:
:thumb:

Reply:
Can you restore the migration guide link as well? I'm hesitant to remove stuff from there since it's where you'd usually go to look for docs, and it's a link to a separate page.

Reply:
👍